config/apply/ccsm/collapse.js

```diff
@@ -165,9 +156,15 @@ export function collapseIntoOne(cards, useHtml=false) {
   return justOneCard;
 }
 
+const table = pug.compile(
+  'table(style={"border-spacing":"0"})\n'+
+  '\teach val in history\n' +
+  '\t\ttr\n'+
+  '\t\t\ttd(style={"border-bottom":"1px solid #eee","padding":"5px 20px 5px 0","width":"8em"}) #{val.date ? dayjs(val.date).format("MM/DD/YYYY") : "n/a"}\n' +
+  '\t\t\ttd(style={"border-bottom":"1px solid #eee","padding":"5px 20px 5px 0","width":"14em"}) #{val.name}\n' +
+  '\t\t\ttd(style={"border-bottom":"1px solid #eee","padding":"5px 20px 5px 0"}) #{val.value || "n/a"}'
```

Comment:
**UI Suggestion**: Should the table columns have headers?

Reasoning:
- If only one history item was present with a value of "n/a", it wouldn't be clear what "n/a" is referring to.
- The third column means something different depending on the resource. For "Test & Labs", it stores the result, For "Procedures", it stores the status, etc.
- There could be different dates associated with a resource, and therefore instances when it is not entirely clear what the date column is referring to.

Reply:
I left the headers off the tables for a couple of reasons:

- Saves 1 line of height for each table in the BPA
- The users are already familiar with the three column format: [ Date | Tests and Procedures | Result ] in many other displays in Epic. 
- The display is easily learned and repetitive - it's always going to be the same layout so users will easily familiarize themselves.
- Our existing card display does not provide headers or labels for the three values currently shown in list format.

I think it might be better to change missing dates to "Date Unknown" - I was finally able to find examples of Epic UI of items without a date and this is what is used.

Reply:
I'm fine with proceeding without the headers. We can always consider adding them down the road based upon clinician feedback.

I agree that the "Date Unknown" text for missing dates will make things a lot more clearer. Do you want to make that change in this PR?

Reply:
Yes, adding "Date Unknown" to this PR

Reply:
Should the result column also default to something more understandable than "n/a"? I just think without a column header, it is not clear what "n/a" would be referring to, unless you have the context of several other results.

Reply:
I think it is fine for now. It is easily learned context - users will see results in the column and associate the type of test with the result. Other patient history display in the Cervical Cancer Screening History display uses "n/a" when the result is not available, without label: 
![Screenshot 2024-04-26 at 12 48 17 PM](https://github.com/ccsm-cds-tools/ccsm-cds-service/assets/526323/26028908-8b25-4cb4-9d5a-99c25d6af72e)

Reply:
Okay. We can consider modifying based upon feedback from pilot users.